helix-view/src/tree.rs

```diff
@@ -529,6 +529,18 @@ impl Tree {
         }
     }
 
+    pub fn focus_view_idx(&mut self, idx: usize) {
+        // If the view doesn't exist, do nothing
+
+        let mut views = self.views().skip(idx);
+
+        let focus_view_id = views.next().map(|view| view.0.id);
```

Comment:
```suggestion
        let focus_view_id = self.views().nth(idx).map(|view| view.0.id);
```

Reply:
[ddress review comments and suggestions](https://github.com/helix-editor/helix/pull/2905/commits/ba97a7d301c31efb626bff23296202edc7a24f1f)